plugin/src/main/java/net/dzikoysk/funnyguilds/config/PluginConfiguration.java

```diff
@@ -1225,7 +1238,8 @@ public void loadProcessedProperties() {
         this.joinItems = this.loadItemStackList(this.joinItems_);
         this.baseItems = this.loadItemStackList(this.baseItems_);
 
-        this.livesRepeatingSymbol = ChatUtils.colored(this.livesRepeatingSymbol_);
+        this.livesRepeatingSymbol.full = ChatUtils.colored(this.livesRepeatingSymbol.full_);
+        this.livesRepeatingSymbol.empty = ChatUtils.colored(this.livesRepeatingSymbol.empty_);
```

Comment:
Może dodać jakiś transformer w stylu ColoredString, bo to już któraś z kolei taka wartość do poprawki

Reply:
Zrobiłem to w hakcy sposób (taki jaki w messages.yml) bo inaczej sie nie da. Ale działa więc git. My w FG lubimy hacky  rzeczy